packages/api/src/api/helix/eventSub/HelixEventSubApi.ts

```diff
@@ -441,7 +441,6 @@ export class HelixEventSubApi extends BaseApi {
 	/**
 	 * Subscribe to events that represent a charity campaign starting in a channel.
 	 *
-	 * @beta
```

Comment:
All these events should also switch versions.

Reply:
Ah yeah, I forget about the version change from beta. Fixed.